modules/system/console/WinterSniff.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace System\Console;
+
+use Illuminate\Support\Facades\File;
+use Symfony\Component\Process\Exception\ProcessSignaledException;
+use Symfony\Component\Process\ExecutableFinder;
+use Symfony\Component\Process\Process;
+use System\Classes\PluginManager;
+
+/**
+ * Console command to check code style and formatting issues.
+ *
+ * If a plugin is provided, this command will search for a `phpcs.xml file inside the plugin's directory. If not found,
+ * the user will be prompted to create one if they wish.
+ */
+class WinterSniff extends BaseScaffoldCommand
+{
+    /**
+     * @var string|null The default command name for lazy loading.
+     */
+    protected static $defaultName = 'winter:sniff';
+
+    /**
+     * @var string The console command name.
+     */
+    protected $name = 'winter:sniff';
+
+    /**
+     * The console command description.
+     */
+    protected $description = 'Validates source code to ensure a consistent coding style.';
+
+    /**
+     * @var string The console command signature as ignoreValidationErrors causes options not to be registered.
+     */
+    protected $signature = 'winter:sniff
+        {paths?* : The path to the files and/or directories to check, if you wish to limit the scope of files checked.}
+        {?--c|config= : Path to a custom PHPCS configuration XML file}
+        {?--p|plugin= : Checks the coding style of a plugin}
+        {?--e|no-warnings : Ignore warnings and only show errors}
+        {?--s|summary : Display a summary of the results}
+    ';
+
+    protected $stubs = [
+        'scaffold/phpcs/phpcs.xml.stub' => 'phpcs.xml',
+    ];
+
+    /**
+     * Create a new command instance.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        // Register aliases for backwards compatibility with October
+        $this->setAliases(['winter:phpcs']);
```

Comment:
We have a property for that already: https://github.com/wintercms/winter/blob/develop/modules/system/console/asset/mix/MixCompile.php#L34-L39